src/uct/base/uct_md.c

```diff
@@ -1,5 +1,5 @@
 /**
-* Copyright (C) Mellanox Technologies Ltd. 2001-2014.  ALL RIGHTS RESERVED.
+* Copyright (C) Mellanox Technologies Ltd. 2001-2021.  ALL RIGHTS RESERVED.
```

Comment:
not sure if it's needed since it's not a new file. @yosefe ?

Reply:
not mandatory but it's ok to add